main.js

```diff
@@ -0,0 +1,200 @@
+const searchButton = document.getElementById("searchButton");
+const inputField = document.getElementById("inputField");
+const flag = document.getElementById("countryFlag");
+const displayCountryInfo = document.getElementById("countryInfo");
+const displayCountryName = document.getElementById("countryName");
+const displayCountryCurrency = document.getElementById("countryCurrency");
+const displayCountryLanguage = document.getElementById("countryLanguage");
+const errorMessage = document.getElementById("errorMessage");
+
+function clearInputField () {
+    inputField.value = null;
+}
+
+inputField.addEventListener("keyup", async (e) => {
+    try {
+        if (e.key === "Enter") {
```

Comment:
Het liefst wil je zo min mogelijk indenting. Dit kun je doen door de inverse te gebruiken van het statement, en zodra dat waar is te returnen:
```
if (e.key !== 'Enter') return;

blablabla hier komt code zonder extra indent
```